src/ResourceManager/RecoveryServices/Commands.RecoveryServices/help/Get-AzureRmRecoveryServicesVaultSettingsFile.md

```diff
@@ -20,6 +20,25 @@ Get-AzureRmRecoveryServicesVaultSettingsFile [-Vault] <ARSVault> -SiteIdentifier
  [<CommonParameters>]
 ```
 
+### ForSiteWithCertificate
```

Comment:
These doc changes should not be added since they should only apply to NetStandard. Docs currently only represent net452 information. We will be making a change as we are transitioning to NetStandard cmdlets only that will generate the docs for NetStandard bits.

Reply:
this cmdlet option  will be for both netStandard  and net452

Reply:
@viverm That's fine if you want to add it to net452. But, you need to remove (#if !NETSTANDARD) the other parameter set (that does not require passing in a cert) from NetStandard, since it won't work for that framework.

Reply:
resolved. Not taking fot net452 in this release